spec/models/pseud_spec.rb

```diff
@@ -48,4 +48,28 @@
       @pseud.errors[:icon_comment_text].should_not be_empty
     end
   end
+
+  describe ".abbreviated_list" do
+    let(:user) { FactoryBot.create(:user, login: "Zaphod") }
+    let(:subject) { user.pseuds.abbreviated_list }
+
+    before do
+      FactoryBot.create(:pseud, user: user, name: "Slartibartfast")
+      FactoryBot.create(:pseud, user: user, name: "Agrajag")
+      FactoryBot.create(:pseud, user: user, name: "Betelgeuse")
+      allow(ArchiveConfig).to receive(:ITEMS_PER_PAGE).and_return(3)
+    end
+
+    it "returns a maximum of ITEMS_PER_PAGE pseuds" do
+      expect(subject.count).to eq(3)
+    end
+
+    it "always contains the default pseud first" do
+      expect(subject.first.name).to eq("Zaphod")
+    end
+
+    it "is in alphabetical order after the default pseud" do
```

Comment:
The last assertion covers the previous two, I think we can remove those and update this description.

Reply:
Fixed in my PR to Cristina, though I personally prefer the more granular tests for helping investigate failures